Objective-C/CBLReplicatorConfiguration.h

```diff
@@ -92,6 +92,19 @@ typedef enum {
  */
 @property (nonatomic, nullable) NSArray<NSString*>* documentIDs;
 
+
+#if TARGET_OS_IPHONE
+/**
+ Allows the replicator to continue replicating in the background. The default
+ value is NO, which means that the replicator will suspend itself when the
+ replicator detects that the application is running in the background.
+ 
+ If setting the value to YES, please ensure that the application requests
+ for extending the background task properly.
```

Comment:
How about "your application delegate requests background time from the OS until the replication finishes".

Actually, is this true? The code seems to take care of requesting a background task on its own, just as 1.x did.

Reply:
If set to YES, the code didn't start the background process. It will be up to the app developer to do that. This is for the app like location based app that may want to have the replicator keeps running in the background without suspending.